tunnistamo/settings.py

```diff
@@ -383,9 +387,123 @@
 SOCIAL_AUTH_ESPOO_ADFS_KEY = ''
 SOCIAL_AUTH_ESPOO_ADFS_SECRET = None
 
-IPWARE_META_PRECEDENCE_ORDER = ('REMOTE_ADDR',)
+# Social Auth Suomi.fi values. These should be overwritten in local settings.
```

Comment:
Should *all* of these be overwritten? For example SOCIAL_AUTH_SUOMIFI_SP_EXTRA ?
Do the test settings contain enough of a documentation (in the form of an example) of what should be configured and how, or would we need a separate documentation?

Reply:
Settings have been reordered and comments are now a bit more verbose.